src/current/cockroachcloud/create-your-cluster.md

```diff
@@ -150,9 +142,24 @@ Otherwise, to enable VPC peering:
       {{site.data.alerts.callout_info}}
       Make sure that you [delete your trial cluster]({% link cockroachcloud/cluster-management.md %}#delete-cluster) before the trial expires. Your credit card will be charged after the trial ends. You can check the validity of the code on the [Billing]({% link cockroachcloud/billing-management.md %}) page.
       {{site.data.alerts.end}}
-1. Click **Create cluster**.
 
-Your cluster will be created in approximately 20-30 minutes.
+## Step 7. Name the cluster
+
+The cluster is automatically given a randomly-generated name. If desired, change the cluster's name. The cluster name must be 6-20 characters in length, and can include lowercase letters, numbers, and dashes (but no leading or trailing dashes). A cluster's name cannot be edited after it is created.
+
+## Step 8. Select the CockroachDB version
```

Comment:
nit: It _may_ be worth calling out that this option may not appear at all if only one version is available. We recently had some internal confusion about this :)

Reply:
Oh, I don't see the option in Prod! When did this change!? What do we mean by "only one version available"?

Reply:
> What do we mean by "only one version available"?

Dedicated clusters default to the latest stable release. We had Pre-Production Preview releases enabled before 23.2 was generally-available and showed users an option to select that version. But now that that version is GA, it's the default and users don't get a version selector option.

Some customers _do_ have that version persistently (e.g. several of our internal orgs), but they have to contact us and we have to enable a feature flag for them.

Reply:
Hmm I think the current text does explain that. Let me know if you disagree (reading through the Slack thread, I do see there was some confusion, but can't figure out what caused that confusion since the text seems pretty clear to me. let me know what I'm missing)

Reply:
I guess the only thing I can think of is to put "(Optional)" at the end of the heading like we do for GCP's VPC peering?

Reply:
@Amruta-Ranade We have some internal-only features enabled in our Prod instance that let us see the drop-down of versions. I don't think this is available to customers for whom we have not enabled that option (I'm not sure we have enabled it for anyone). If you use the Education Prod org, you should see the experience a customer would see. @sjbarag am I incorrect here?

Reply:
Confirmed on Slack: The docs are correct!